### PR TITLE
surbhirathi.json

### PR DESCRIPTION
**Pull request by SurbhiRathi23**
added myself to hacktoberest-particants list

### CODE REVIEW

File: src/contributors/surbhirathi.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+  
+{
+    "name": "Surbhi Rathi", 
+    "desc": "Web Developer",
+    "github": "https://github.com/SurbhiRathi23/",
```
Comment (iamdarshshah) [inline src/contributors/surbhirathi.json:5]:
```suggestion
    "github": "https://github.com/SurbhiRathi23/"
```
Remove this trailing comma.

**Review by iamdarshshah — CHANGES_REQUESTED** [review]
Your file name should be `YOUR-GITHUB-USERNAME.json`. Please, update your file-name.

**Comment by iamdarshshah** [comment]
@SurbhiRathi23 check again, it seems you have deleted your file.

**Comment by SurbhiRathi23** [comment]
I made a new pull request

**Pull request closed without merging**